src/data/actions/grades.js

```diff
@@ -194,9 +194,9 @@ const fetchGrades = (
 );
 
 const formatGradeOverrideForDisplay = historyArray => historyArray.map(item => ({
-  date: formatDateForDisplay(new Date(item.history_date)),
-  grader: item.history_user,
-  reason: item.override_reason,
+  date: formatDateForDisplay(new Date(item.created)),
```

Comment:
@luismorenolopera these changes are related to the old old names of the fields in Ironwood?

Reply:
Yes, and earned_graded_override that is being created in the other PR.